src/www/diag_logs_filter.php

```diff
@@ -745,7 +745,7 @@ function getGETPOSTsettingvalue($settingname, $default)
 									  <td class="listMRr nowrap"><?php echo htmlspecialchars($filterent['time']);?></td>
 									  <td class="listMRr nowrap">
 										<?php if ($filterent['direction'] == "out"): ?>
-										<span class="glyphicon glyphicon-cloud-download" alt="Direction=OUT" title="Direction=OUT"></span>
+                    <span class="glyphicon glyphicon-cloud-download" alt="<?= gettext('Direction=OUT') ?>" title="<?= gettext('Direction=OUT') ?>"></span>
```

Comment:
This looks more like programming than something that is easy to read or to translate! Either we drop this translation or clean it up further?